ansible/roles/test/files/ptftests/fib_test.py

```diff
@@ -239,19 +240,19 @@ def check_balancing(self, dest_port_list, port_hit_cnt):
         logging.info("%-10s \t %10s \t %10s \t %10s" % ("port(s)", "exp_cnt", "act_cnt", "diff(%)"))
         result = True
 
-        total_hit_cnt = float(sum(port_hit_cnt.values()))
+        total_hit_cnt = sum(port_hit_cnt.values())
         for ecmp_entry in dest_port_list:
-            total_entry_hit_cnt = 0.0
+            total_entry_hit_cnt = 0
             for member in ecmp_entry:
                 total_entry_hit_cnt += port_hit_cnt.get(member, 0)
-            (p, r) = self.check_within_expected_range(total_entry_hit_cnt, total_hit_cnt/len(dest_port_list))
+            (p, r) = self.check_within_expected_range(total_entry_hit_cnt, float(total_hit_cnt)/len(dest_port_list))
             logging.info("%-10s \t %10d \t %10d \t %10s"
                          % (str(ecmp_entry), total_hit_cnt/len(dest_port_list), total_entry_hit_cnt, str(round(p, 4)*100) + '%'))
             result &= r
-            if len(ecmp_entry) == 1:
+            if len(ecmp_entry) == 1 or not total_entry_hit_cnt:
```

Comment:
total_entry_hit_cnt is an integer, so it is better to use (total_entry_hit_cnt == 0). "not" is used for bool.